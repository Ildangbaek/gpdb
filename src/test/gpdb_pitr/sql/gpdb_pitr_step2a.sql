-- Step 2

CREATE TABLE gpdb_pitr_tt(i INT);
INSERT INTO gpdb_pitr_tt VALUES (1), (2), (3);
