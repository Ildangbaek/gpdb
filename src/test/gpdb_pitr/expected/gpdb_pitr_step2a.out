-- Step 2
CREATE TABLE gpdb_pitr_tt(i INT);
NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'i' as the Greenplum Database data distribution key for this table.
HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
INSERT INTO gpdb_pitr_tt VALUES (1), (2), (3);
